src/content.js

```diff
@@ -174,11 +188,44 @@ function apply(options, insertionPoint) {
 	}
 }
 
+function requestPage(number) {
+	return fetch(`https://github.com/dashboard/index/${number}?utf8=%E2%9C%93`, {
+		credentials: 'include',
+		headers: {
+			'X-Requested-With': 'XMLHttpRequest'
+		}
+	})
+		.then(data => data.text());
+}
+
+function preloadPages(options) {
+	Promise.all(
+		Array(options.preloadPagesCount - 1)
+			.fill(null)
+			.map((t, n) => requestPage(n + 1))
+	)
```

Comment:
Ok we can leave this custom fetching method, but I'd still like them to display as soon as possible, instead of waiting for all of them to load.

I'm not sure of the exact method, but:

* load them all at once like you're doing
* append the them as they come **but** maintain the correct order.

One of these functions will likely do that: https://github.com/sindresorhus/promise-fun